upstream-community-operators/traefikee-operator/0.1.1/traefikee-operator.v0.1.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,327 @@
HjNA5An0uyFt69xwB9tuK1eHV1LqXsOEoHTl/gJt069fFryq+nhaDPHPyM1eMI865e/Su99dZbZG9nJy4cPPgCAuCH/L7fGPTVBYB5sbGxHYzrjJKSkpKSlUj25xLg86xhukar7QVoKRDsryfqDxs2jL+269OHE/g//XQ0pWdk0ukzZ005X7KCuKFNphsCg5VmnsG9p4oZBt/8dUcNDM7awBXFtokaCMrm061tRrH8fWVjZjdArbht9PQhNI4QOULlCJmX6HmaDGMVNa+bJSt4jWFeWdRRWnqFq3kT5s6lxx9/nHp070bBwUF8wQD7DeT3NcQqJQCK16I91hcFgEpKSkpWJgmAYrF+TDZR7v8bB0BpMnQHzyhAEEAILw82/RdeeIEWL1lKR44eY/Aj3SsIWLhREJTwcblOGKyg3YXnaa4AoRGrc8lGholnpbFnEOCEcWaAQTsrh0H5uznrY/Tg7XvYUMjxiPi/53/ZRnO2HaadBee4eEYKeZR4bfAaWaKYw/i6y/fPCPjniy/QppzN9O2331JwUBD16NGd+vbty+cFQFCeLy19zlqJyT6AB8Rn526sLwoAlZSUlKxM0dHRf8ZRbF4RWLz1nmQVVrCJWJXJ8DDCe0888QSFhoZS54cf4uKRyZOnUO6OndzyQ3qKZHiYYaIJvQWNdoVhsFyDQUMByQUBRLknz9JsAUivrNqu5QzO0mAQxREAKk8GqywGLhsrgEFb/feAp9Jdn73LEzn06Sl3CgActGwLTdx0gHKOn6bzpWWmvxc9FjXoK7dYXt9V4FdRM6kD958sKKQ1a9ZSfHw82dn14dzQgeIzglGDCPf+BvP7GmIVekeB/QoAlZSUlKxUw4YN+wuOAm68xYJdpQDw2iY3e3h9AAEAwJ49epCHuzuP9ErbkE6FhUU1kx+IanmVbhRUJAyigKTa2HS6VGs6PXXzQXr65230YGKmNttWACHmEwO0AIOYiHGzQVBCH27DM+klfg8ncfs2fUoKpqXELcmhrzK0UWwFFy9TdXVN25YrzQR9RvArN3pZS6/Qgbw8Sk5Ooaeffpq6dnmEvL28ODcUHj+Z36fAr16TIeD94vNxF9YXBYBKSkpKViYDADoLu6IAsGEmAQAbHUAQRxcXF+rZswe99NJLtHDRIjp46DCVXqnxYAEy8LUlQLAWDGK+bFUNDJ66VEpZR0/RhI0H2Jv2R+TSzUij36NqVoCXjwAwt5Tm9wpK8HPQPZGwznN16JuVRq7zsil+3S76eV8+HTt3iecqGyt42evZDNDHrx1DdLkpzIt/EebdunUbTZw0icO78PDC0wuPLzy/8ABjikxLn3utwEwewMGDBysAVFJSUrJGyRDwwIED+4gN7oICwMYZIBBQAK8QvENDhg6lwMBAeqTzwwwMP02eTFu3bWe4ME0aqaoyAYhFvFgVlabQKDexrpYzbavpRHEJrck7SV9s2EMhCzdpIWIBgw8KKIQnzjtFqyK2JAjKx0Lo2XdeFrmI239EiHdGKt0xN4OeM+T1XSozL+Yo1/L6mgH6uC2MAHC87qZqXjRtLiikdevX0yeffsotgFD5HRMTw2Cv8vuaZBV6+xuVA6ikpKRkrZJFIGKx7i7sjB66UUUgTTCAoGwjAxiMiorighF/Pz8OD69dt45zyozFBYARQEm5pXrVGWDQ2FamVNxGy5RFu49xW5kHBJQByOCJcxa3/eZlM7DZGMK1TQE/HOHpA/h1R0EHnmNmGoUu2kTjs/ZRxpEi9lBKwXPZXHl95uBXZqgcxtd5Bw/RgoWL6JVXRpCtjQ05OzlxDqzK77thq5RFIKoKWElJSclKJQFw4MCBD4mF+6QCwBs3CQ6oGgZMAAqdnBy5iGDEiFcZOvIOHjRVD0MSBC0VHpbgIyuJqwz5gucul3GBBULE0YtzGAIBamg07S9AEMUZjQHB3vrRW0Afwsu3z9lA7aan0l8EAKJ1y/ydWr8+Y5Pm8vLKZg3xyjAvw7UBhNG0edv2XJoyZSpDXvdu3cjPz5c9fYB2ACBAXoHfDZlqA6OkpKRk7ZILM2Z2igX7iAJAy1ldbWR8fX2pV88eDBxTp05jGLl4qcQEKEZvlUVhqPzqEDGA7PDZi7R0z3F6/ddcaodcwenrqbM4BggQ9NBBsHdSTcsWafJ+tJrxEd8L8PvdbA383Odn09eZmrfvdMkV099WyVW8WjWzJVu31Ad+xvy+olOnKW3DBhozZgzn9eE9CA8Pp6Gc3zdU5fdZ1lQjaCUlJSVrlxwFJyDl32LB3q8A0PImQVBrIzNUnzISSja9e1NERASNG/cVpadncHNp84IRS+UJmoeItZYyNV7BswLUMgWwfZG+hx5CeHjaem4lAxCU4WGMYkMVMY7wEgYK0EMOIecVCvALW5xDs7Ye4gkmpeWVtbx9lhzJVp/h9YIZp3UcOXKUFi9ZQm+NHEleXl7kYG/PldtPMJCr/L5mMtMoOAmAahSckpKSkpVp2LBhf8QxLi7un2LB3qkAsHnN2EYGlaWoNgWUIAw5atQoWvHzLzx72FQwIiAGbUksDYLSKwgwA6BJr2CJ+Hp7/ln24HFfQQGCyOHrk5hBNonp1CVhA7dweYi9halsUUtyKDH3CHsTK/VK3iq9fUtze/uMhR3G/D6M7Zs9ew4988wz5OhgTx4eHgzgqOj9jc7nvZlWqWYBKykpKVm5JACK49/FBrlNb9/wm5wFfDNNAog2O3Yo33Z3dyM3N1caMWIEpcybTwcPHeIqVQ6fimNzgaD0ChrbyZTqE0fQV3D4z9u08HBCBvWet4nDxE7zN9Lba3bQkt3HuYVLlfw9K2+Ot4/zG8XrYSzsKC6+QBs35dD33/9AAwc+Sg4C/AICAkyvMULxCvxuikkP4CEFgEpKSkpWKgmAYrH+m7CtCgBvrhnzBAEpaCfj6+tDzs5OXEAyfcYM2r17j8nD1ZwgKL2CRhCEB/JkcQntKSqm1bsO0/gVG2hnYTHtLjhHxaVlpobNNw
```

Comment:
please remove "replaces" line as it tries to replace itself. Just delete the line. In next PR the same line will be needed as it replaces the latest public operator.